packages/ui/core-components/src/lib/unsorted/viz/table/GroupRow.svelte

```diff
@@ -50,14 +51,23 @@
 				: column_format}
 		{@const useFormat = format?.valueType === 'date' ? '' : format}
 		{#if j === 0}
-			<TableCell class="font-medium py-[3px]" colSpan={rowNumbers ? 2 : 1} paddingLeft="1px">
+			<TableCell
+				class="{compact ? 'text-xs py-[1px] px-[4px]' : 'py-[2px] px-[8px]'} font-medium py-[3px]"
+				colSpan={rowNumbers ? 2 : 1}
+				paddingLeft="1px"
+			>
```

Comment:
Lot of repetition here passing the compact classes down to `TableCell`, we should pass the compact prop through and handle it in the cell